wscript

```diff
@@ -54,6 +54,8 @@ def configure(conf):
             '/usr/include/SuperCollider/common',
             '/usr/local/include/SuperCollider/plugin_interface',
             '/usr/local/include/SuperCollider/common',
+	    '/home/we/supercollider/external_libraries/nova-simd',
```

Comment:
this include path is stupid.

Reply:
ugh, supercollider doesn't install these files indeed. i think submodules will work great here, these libs are header-only.